src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -71,6 +71,22 @@ message Image {
     string digest = 3;
 }
 
+message NetworkConfiguration {
+    enum NetworkMode {
+        // Provides no network access to a container (Not supported yet)
+        None = 0;
```

Comment:
Proto enum best practices suggest making an "UNKNOWN" field to be the first enum value, for compatibility reasons.  NONE should be field number 1.

Reply:
Here is an example of a canonically structured proto enum:

https://github.com/grpc/grpc/blob/53c72c5936b98015233bc4c86080d8789fd9d9bf/src/proto/grpc/channelz/channelz.proto#L86

Reply:
Ah yes, that looks more familiar